doc/api/tracing.md

```diff
@@ -5,12 +5,9 @@
 Trace Event provides a mechanism to centralize tracing information generated by
```

Comment:
It doesn't have to be in this PR, but we really need to rename this file to `trace_events.md` and give it a correct title :D

Reply:
Yes, definitely :-) ... I can mark that as a todo for another PR.